config/deploy.rb

```diff
@@ -21,7 +21,7 @@
 set :use_sudo, false
 set :pty, true
 set :ssh_options, forward_agent: true
-set :keep_releases, 5
+set :keep_releases, 15
```

Comment:
`5` should be plenty here. Any reason for `15`?

Reply:
Oh yes, i'll change that back to 5. I was just testing and troubleshooting deployment